app/controllers/tasks/collection_objects/filter_controller.rb

```diff
@@ -47,22 +47,60 @@ def set_otu
     render json: {html: collection_objects.count}
   end
 
+  # GET
   def set_id_range
-    render json: {html: collection_objects.count.to_s}
+    render json: {html: collection_objects.count}
+  end
+
+  # GET
+  def set_user_date_range
+    params[:user] = params[:user].to_i
```

Comment:
This seems odd, why here?

Reply:
If not here, it complicates things a bit down line, but I fixed that.